contrib/win32/openssh/keygen.vcxproj

```diff
@@ -264,6 +270,9 @@
       <EntryPointSymbol>wmainCRTStartup</EntryPointSymbol>
       <AdditionalOptions>/debug /debugtype:cv,fixup /opt:ref /opt:icf /incremental:no %(AdditionalOptions)</AdditionalOptions>
     </Link>
+    <Manifest>
+      <AdditionalManifestFiles>targetos.manifest</AdditionalManifestFiles>
```

Comment:
@NoMoreFood - Where did you get this information?

As per [microsoft documentation page](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file),
![image](https://user-images.githubusercontent.com/23668037/70969757-74aa9e80-2051-11ea-8fd6-9b71279a29be.png)